### PR TITLE
Persist simple lists

### PR DESCRIPTION
**Pull request by audiodude**
This rather large PR includes the logic that is called from the backend simple list "create" endpoint. The new logic method `persist_simple_list` handles storing a simple list selection in both the database (for its metadata) and the backend s3 storage (for its article list). Includes tests for the endpoint as well as the logic methods and the model methods.

### CODE REVIEW

**Comment by audiodude** [comment]
@mahakporwal02 sorry for the rather large PR!

File: wp1/logic/selection.py [hunk 1]
```diff
@@ -29,3 +35,68 @@ def validate_list(items):
...
+    cursor.execute(
+        '''INSERT INTO
+           selections (s_name, s_user_id, s_project, s_model,
+                       s_last_generated, s_created_at)
+           VALUES
+           (%(s_name)s, %(s_user_id)s, %(s_project)s, %(s_model)s,
+            %(s_last_generated)s, %(s_created_at)s)
+    ''', attr.asdict(selection))
+    return cursor.lastrowid
+  wp10db.commit()
```
Comment (mahakporwal02) [inline wp1/logic/selection.py:51]: Shouldn't this line should be above return statement?

File: wp1/web/app.py [hunk 2]
```diff
@@ -125,6 +125,7 @@ def swagger_api_docs_yml():
 
   app.register_blueprint(projects, url_prefix='/v1/projects')
   app.register_blueprint(articles, url_prefix='/v1/articles')
+  app.register_blueprint(selection, url_prefix='/v1/selection')
```
Comment (mahakporwal02) [inline wp1/web/app.py:128]: Blueprint has been registered twice.

File: wp1/web/selection.py [hunk 3]
```diff
@@ -14,15 +19,26 @@ def create():
...
         }
-    }
-  return {"success": True, "items": {}}
+    })
+
+  # All valid, time to upload to s3 and store in the application database.
+  s3 = get_storage()
+  wp10db = get_db('wp10db')
+  user_id = flask.session['user']['identity']['sub']
+  logic_selection.persist_simple_list(wp10db, s3, list_name, user_id, project,
```
Comment (mahakporwal02) [inline wp1/web/selection.py:41]: Shouldn't this should be under try catch?

**Review by mahakporwal02 — CHANGES_REQUESTED** [review]
Some changes I could figure out. Will you also add developers overlay afterwards?

**Comment by audiodude** [comment]
Thanks for the review! I'm actually going to abandon this PR based on the comments on #370 and #374. We need a different schema for list builders/selections and a more generalized way of "materializing" built lists to the storage backend as selections.

**Pull request closed without merging**